script/cibuild

```diff
@@ -10,7 +10,13 @@ ruby -v
 echo "hostname: $(hostname)"
 echo "pwd: $(pwd)"
 git --version
+echo "node: $(node --version)"
+echo "npm: $(npm --version)"
 
 script/bootstrap
 
 bundle exec rake test
+
+find ./_site -type d \( -name assets -o -name _javascript -o -wholename downloads/es_ES/github-git-cheat-sheet/index.html \) -prune -o -name "*.html" -print >> ./script/list-of-files.txt
```

Comment:
@brianamarie probably there are multiple solutions... One way would be to manually add them to the end of the  `list-of-files.txt`, after the `find` operation e.g.

```bash
echo foo >> list-of-files.txt
echo bar >> list-of-files.txt
```